### DISCUSSION TITLE
39 RECAP go to 19 RECAP

### DISCUSSION THREAD

**Discussion by pattool** [Cyfrin/moccasin-full-course-cu#12] [Q&A]
When I want to play the 39 Recap of Section 1 Python in Updraft it plays the 19 Recap instead of the 39 one. Any suggestion?

**Comment by Scofield-Idehen** [comment]
You can check your network connection if that is ok, then I suggest you open Cyfrin on a new tab or browser, it works for me I believe you are having a network issue. 

Best.

**Reply by pattool** [reply]
It is solved like you say with a new incognito window. Thank you for your help.

**Reply by EngrPips** [reply]
Interesting.
